paperless/downloadfile.go

```diff
@@ -0,0 +1,80 @@
+package paperless
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var documentselection int
+
+func (p Paperless) setFullDownloadURL(location string) string {
```

Comment:
Small bug in this code too where it fails to take into account a non-standard port other than 80 (but just follow the suggestion below, instead).

Reply:
👍